frontend/src/app/features/book/components/book-menu/book-menu.component.html

```
@@ -0,0 +1,124 @@
+  }
+  @if (openInNewTab() && book(); as menuBook) {
+    <app-menu-item [link]="['/book', menuBook.id]" target="_blank" [trailingIcon]="newTabIcon">{{ 'book.menu.openInNewTab' | transloco }}</app-menu-item>
+    <app-menu-separator />
+  }
+  <app-menu-item [submenu]="shelfMenu.ariaMenu()">{{ 'book.menu.addToShelf' | transloco }}</app-menu-item>
+  <app-menu-item [submenu]="markAsMenu">{{ 'book.menu.markAs' | transloco }}</app-menu-item>
+  @if (actionPermissions().canEmailBook && digital()) {
+    <app-menu-item [submenu]="sendMenu">{{ 'book.menu.send' | transloco }}</app-menu-item>
+  }
+  @if (actionPermissions().canDownload && digital()) {
+    @if (additionalFiles().length > 0) {
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)